Dockerfile

```diff
@@ -9,8 +9,6 @@ RUN apt-get install -q -y --no-install-recommends software-properties-common git
 RUN add-apt-repository ppa:deadsnakes/ppa
 RUN apt-get update
 RUN apt-get install -q -y --no-install-recommends \
-    python3.6 \
-    python3.7 python3.7-distutils \
```

Comment:
Like this? Is there another Dockerfile I'm missing?

Reply:
yep, this file